tensorboard/backend/http_util_test.py

```diff
@@ -205,6 +206,18 @@ def testCsp_noHash(self):
     )
     self.assertEqual(r.headers.get('Content-Security-Policy'), expected_csp)
 
+  def testCsp_disableStrictDynamic(self):
+    http_util.DO_NOT_USE_CSP_SCRIPT_HASHES_STRICT_DYNAMIC = False
```

Comment:
Prefer mock.object.patch() for this since it can be auto-restored after the test.  E.g. you can just use it as as decorator on the test method.

Note we need a conditional import for it:
```
try:
  # python version >= 3.3
  from unittest import mock  # pylint: disable=g-import-not-at-top
except ImportError:
  import mock  # pylint: disable=g-import-not-at-top,unused-import
```

Reply:
It was unclear how I would mock out module global like the ones I declared so I converted them to methods. Please advise on how I can better improve this code :)